client/src/components/container/admin/UsersPage/index.js

```diff
@@ -0,0 +1,80 @@
+import React,{Component} from "react";
+import ReactHTMLTableToExcel from 'react-html-table-to-excel';
+import Axios from "axios";
+import { Button, Empty } from "antd";
+import {LoadingOutlined} from "@ant-design/icons"
+
+import "./style.css"
+
+class UsersPage extends Component {
+    state={
+        users:[],
+        isLoaded:false,
+        error:null
+    }
+    componentDidMount(){
+        const {match:{params:{eventcode}}}=this.props
```

Comment:
There's a lot happening in this file, I would split it into two or three React components. You can have a parent component that loads the data and then makes a decision of which child component to display based on the result, passing whatever the child needs as props